build.sbt

```diff
@@ -28,10 +28,10 @@ libraryDependencies ++= {
     "com.typesafe.play" %% "play-ws"          % playVersion,
     "org.slf4j"         % "slf4j-api"         % "1.7.21"
   ) ++ Seq(
-    "org.scalatest"            %%  "scalatest"       % "2.2.6"     % "test->*",
```

Comment:
I think the correct configuration to use for that dependency is "test", I'm not sure where the ->* came from (I added it apparently).